src/index.html

```diff
@@ -4,10 +4,16 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Breaking News</title>
+    <title>Recipes</title>
 </head>
 <body>
-    <h1>Hello world!</h1>
     
+    
+
+    <footer>
+        <div class="footer">
+            <p class="rights">Created by <a class="developer" href="https://github.com/devMahmoud">Mahmoud Makhlouf</a> & <a class="developer" href="https://github.com/marurevi">Mariana Revilla</a></p>
+        </div>
+    </footer>
```

Comment:
- A good solution for the width in the footer. However, I think this `div` is now obsolete. Try to remove it if it's possible.